react.gradle

```diff
@@ -21,7 +21,26 @@ def detectEntryFile(config) {
     return "index.js";
 }
 
-def cliPath = config.cliPath ?: "node_modules/react-native/cli.js"
+/**
+ * Detects CLI location in a similar fashion to the React Native CLI
+ */
+def detectCliPath(config) {
+    if (config.cliPath) {
+        return config.cliPath
+    }
+
+    def cliPath = ["node", "-e", "console.log(require('react-native/cli').bin);"].execute([], projectDir).text
```

Comment:
Does this mean that we have to have `react-native` installed as a global dependency?

Reply:
Nope,  we're running this from `projectDir` context, which is a directory inside your project. Will resolve to local `node_moduls`.

Reply:
Can we connect somewhere in a bit more "direct" way to quickly resolve this? If you're on Twitter or Discord, we can exchange DMs, put few console.logs here and there and see what's going on.

I'm @grabbu or grabbou@gmail.com - will be online for next two hours. Let's try tackling that!